custom_components/tahoma/climate_devices/atlantic_pass_apc_heating_zone.py

```diff
@@ -0,0 +1,260 @@
+)
+COMMAND_REFRESH_DEROGATION_REMAINING_TIME = "refreshDerogationRemainingTime"
+COMMAND_REFRESH_ECO_HEATING_TARGET_TEMPERATURE = "refreshEcoHeatingTargetTemperature"
+COMMAND_REFRESH_PASS_APC_HEATING_MODE = "refreshPassAPCHeatingMode"
+COMMAND_REFRESH_PASS_APC_HEATING_PROFILE = "refreshPassAPCHeatingProfile"
+COMMAND_REFRESH_TARGET_TEMPERATURE = "refreshTargetTemperature"
+COMMAND_SET_COMFORT_HEATING_TARGET_TEMPERATURE = "setComfortHeatingTargetTemperature"
+COMMAND_SET_DEROGATION_ON_OFF_STATE = "setDerogationOnOffState"
+COMMAND_SET_DEROGATED_TARGET_TEMPERATURE = "setDerogatedTargetTemperature"
+COMMAND_SET_DEROGATION_TIME = "setDerogationTime"
+COMMAND_SET_ECO_HEATING_TARGET_TEMPERATURE = "setEcoHeatingTargetTemperature"
+COMMAND_SET_OPERATING_MODE = "setOperatingMode"
+COMMAND_SET_PASS_APC_HEATING_MODE = "setPassAPCHeatingMode"
+COMMAND_SET_TARGET_TEMPERATURE = "setTargetTemperature"
+
+CORE_COMFORT_HEATING_TARGET_TEMPERATURE_STATE = (
+    "core:ComfortHeatingTargetTemperatureState"
+)
+CORE_DEROGATED_TARGET_TEMPERATURE_STATE = "core:DerogatedTargetTemperatureState"
+CORE_DEROGATION_ON_OFF_STATE = "core:DerogationOnOffState"
+CORE_ECO_HEATING_TARGET_TEMPERATURE_STATE = "core:EcoHeatingTargetTemperatureState"
+CORE_HEATING_ON_OFF_STATE = "core:HeatingOnOffState"
+CORE_TARGET_TEMPERATURE_STATE = "core:TargetTemperatureState"
+
+IO_DEROGATION_REMAINING_TIME_STATE = "io:DerogationRemainingTimeState"
+IO_PASS_APC_HEATING_MODE_STATE = "io:PassAPCHeatingModeState"
+IO_PASS_APC_HEATING_PROFILE_STATE = "io:PassAPCHeatingProfileState"
+IO_TARGET_HEATING_LEVEL_STATE = "io:TargetHeatingLevelState"
+
+PASS_APC_HEATING_MODE_STATE_ABSENCE = "absence"
+PASS_APC_HEATING_MODE_STATE_COMFORT = "comfort"
+PASS_APC_HEATING_MODE_STATE_DEROGATION = "derogation"
+PASS_APC_HEATING_MODE_STATE_INTERNAL_SCHEDULING = "internalScheduling"
+PASS_APC_HEATING_MODE_STATE_STOP = "stop"
+PASS_APC_HEATING_PROFILE_STATE_ABSENCE = "absence"
+PASS_APC_HEATING_PROFILE_STATE_COMFORT = "comfort"
+PASS_APC_HEATING_PROFILE_STATE_DEROGATION = "derogation"
+PASS_APC_HEATING_PROFILE_STATE_ECO = "eco"
+PASS_APC_HEATING_PROFILE_STATE_INTERNAL_SCHEDULING = "internalScheduling"
+PASS_APC_HEATING_PROFILE_STATE_STOP = "stop"
+
+MAP_PRESET_MODES = {
+    PASS_APC_HEATING_PROFILE_STATE_ECO: PRESET_ECO,
+    PASS_APC_HEATING_PROFILE_STATE_COMFORT: PRESET_COMFORT,
+    PASS_APC_HEATING_PROFILE_STATE_INTERNAL_SCHEDULING: CUSTOM_PRESET_AUTO,
+    PASS_APC_HEATING_PROFILE_STATE_DEROGATION: CUSTOM_PRESET_DEROGATION,
+    PASS_APC_HEATING_PROFILE_STATE_STOP: CUSTOM_PRESET_STOP,
+    PASS_APC_HEATING_PROFILE_STATE_ABSENCE: PRESET_AWAY,
+}
+MAP_REVERSE_PRESET_MODES = {v: k for k, v in MAP_PRESET_MODES.items()}
+
+
+class AtlanticPassAPCHeatingZone(OverkizEntity, ClimateEntity):
+    """Representation of Atlantic Pass APC Heating and Cooling Zone."""
+
+    _attr_hvac_modes = [HVAC_MODE_OFF, HVAC_MODE_HEAT, HVAC_MODE_AUTO]
+    _attr_max_temp = 30
+    _attr_min_temp = 5
```

Comment:
These values are not exposed by Atlantic?

Reply:
not in this device (device number 7)

the navilink sensor (device number 8) have 2 attributes (from 5 to 30) :
```
2022-02-20 21:24:28 DEBUG (MainThread) [custom_components.tahoma] Unsupported device detected (Device(attributes=States(_states=[.... State(name='core:MaxSensedValue', type=<DataType.FLOAT: 2>, value=30.0), State(name='core:MinSensedValue', type=<DataType.FLOAT: 2>, value=5.0)]),available=True, enabled=True, label=** *(**#**)*, device_url=io://****-****-0589/12309249#8, controllable_name='io:AtlanticPassAPCZoneTemperatureSensor' ....
```

tahomalink allows to set a temperature between 7 and 35 ...